### PR TITLE
Update Callisto Network

### PR DESCRIPTION
**Pull request by ESNJS**
@ligi I opened a new PR so I do not spam the branch with commits.

### CODE REVIEW

**Review by ligi — CHANGES_REQUESTED** [review]
name must be unique - but found `Callisto Testnet` more than once
so please rename the deprecated one

**Comment by ESNJS** [comment]
> name must be unique - but found `Callisto Testnet` more than once so please rename the deprecated one

@ligi done, I hope haha

File: _data/chains/eip155-821.json [hunk 1]
```diff
@@ -11,5 +11,6 @@
   "infoURL": "https://callisto.network",
   "shortName": "tclo",
   "chainId": 821,
-  "networkId": 2
```
Comment (SQuIsHY7D3m) [inline _data/chains/eip155-821.json:14]: -

Comment (SQuIsHY7D3m) [inline _data/chains/eip155-821.json:14]: #1411

**Pull request merged**